test/k8s/services.go

```diff
@@ -416,6 +416,27 @@ Secondary Interface %s :: IPv4: (%s, %s), IPv6: (%s, %s)`,
 			}
 		})
 
+		Context("with L7 policy", func() {
```

Comment:
Is it possible to test within the Cilium CLI connectivity test or do we need to add more tests to Ginkgo here?

Reply:
Commit dropped, because I just released this e2e test is already there at https://github.com/cilium/cilium/blob/ca5a9d04b55b6dbb08d6b8d3c1b4a96f78f7df84/test/k8s/services.go#L108